parca.yaml

```diff
@@ -11,6 +11,11 @@ debug_info:
 scrape_configs:
   - job_name: "default"
     scrape_interval: "3s"
-    scrape_timeout: "2s"
     static_configs:
       - targets: [ '127.0.0.1:7070' ]
+    profiling_config:
+      pprof_config:
+        fgprof:
+          enabled: true
+          path: /debug/pprof/fgprof
```

Comment:
We shouldn't add this to the default scrape config. Verrry few apps will have this.